Kickstarter-iOS/Views/Controllers/DiscoveryFiltersViewController.swift

```diff
@@ -9,26 +9,29 @@ internal protocol DiscoveryFiltersViewControllerDelegate: class {
 }
 
 internal final class DiscoveryFiltersViewController: UIViewController, UITableViewDelegate {
-  @IBOutlet fileprivate weak var closeButton: UIButton!
-  @IBOutlet fileprivate weak var backgroundGradientView: GradientView!
-  @IBOutlet fileprivate weak var filtersTableView: UITableView!
+  @IBOutlet private weak var backgroundGradientView: GradientView!
+  @IBOutlet private weak var closeButton: UIButton!
+  @IBOutlet private weak var filtersTableView: UITableView!
 
-  fileprivate let dataSource = DiscoveryFiltersDataSource()
-  fileprivate let viewModel: DiscoveryFiltersViewModelType = DiscoveryFiltersViewModel()
+  private let activityIndicator = UIActivityIndicatorView()
+  private let dataSource = DiscoveryFiltersDataSource()
+  private let viewModel: DiscoveryFiltersViewModelType = DiscoveryFiltersViewModel()
 
   internal weak var delegate: DiscoveryFiltersViewControllerDelegate?
 
-  internal static func configuredWith(selectedRow: SelectableRow, categories: [KsApi.Category])
+  internal static func configuredWith(selectedRow: SelectableRow)
     -> DiscoveryFiltersViewController {
 
       let vc = Storyboard.Discovery.instantiate(DiscoveryFiltersViewController.self)
-      vc.viewModel.inputs.configureWith(selectedRow: selectedRow, categories: categories)
+      vc.viewModel.inputs.configureWith(selectedRow: selectedRow)
       return vc
   }
 
   override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.view.addSubview(activityIndicator)
```

Comment:
we chatted about this irl, but for posterity we decided it might better to put the activity indicator in a cell so that we don't have to worry about it's positioning with respect to the top filters